core/impl/pom.xml

```diff
@@ -50,11 +50,11 @@
             <artifactId>blaze-common-utils</artifactId>
         </dependency>
 
-        <!-- Actually we only need JPA but since there is no official artifact 
-            we rather use this than provider specific apis -->
+        <!-- JPA 2.2 API for getResultStream signature -->
         <dependency>
-            <groupId>org.apache.openejb</groupId>
-            <artifactId>javaee-api</artifactId>
+            <groupId>javax.persistence</groupId>
```

Comment:
Let's use the following dependency instead and please declare it in the parent/pom.xml like the `javaee-api` one so that we can omit the version here:
```
<dependency>
  <groupId>jakarta.persistence</groupId>
  <artifactId>jakarta.persistence-api</artifactId>
  <version>2.2.3</version>
</dependency>
```